### PR TITLE
Mobile 2775 Reduce nuget path longth

### PR DESCRIPTION
**Pull request by khmMouna**
### What do these changes do?
Reduce the nuget package name as possible and use Airship instead of Urbanairship

### Why are these changes necessary?
Fix an issue with windows nuget long path.

### How did you verify these changes?
Run sample app and test manually.

### CODE REVIEW

File: Airship.iOS.Basement.nuspec [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
    <files>
      <file src="src/AirshipBindings.iOS.Basement/bin/release/AirshipBindings.iOS.Basement.dll" target="lib/Xamarin.iOS10" />
-     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/UABasement.resources" />
-     <file src="src/AirshipBindings.iOS.Basement/urbanairship.ios.basement.targets" target="build/urbanairship.ios.basement.targets" />
+     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/ABasement.resources" />
```
Comment (rlepinski) [inline Airship.iOS.Basement.nuspec:18]: This seems to be inside our module right? Can we just say `content/res.resources` ?

**Review by rlepinski — APPROVED** [review]
Looks good, wonder if we can just normalize on a resource path though

File: Airship.iOS.Accengage.nuspec [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
    <files>
      <file src="src/AirshipBindings.iOS.Accengage/bin/release/AirshipBindings.iOS.Accengage.dll" target="lib/Xamarin.iOS10" />
-     <file src="src/AirshipBindings.iOS.Accengage/bin/Release/AirshipBindings.iOS.Accengage.resources/**" target="content/UAAccengage.resources" />
```
Comment (rlepinski) [inline Airship.iOS.Accengage.nuspec:18]: I think we can just do `content/res` for all of them

**Comment by rlepinski** [comment]
Is it possible to also rename the xcframeworks that we are consuming? AirshipPreferenceCenter.xcframework is pretty long

**Review by oristanovic — APPROVED** [review]

**Pull request merged**